src/Common/Http/ResponseInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * SocialConnect project
+ * @author: Patsura Dmitry @ovr <talk@dmtry.me>
+ */
+
+namespace SocialConnect\Common\Http;
+
+interface ResponseInterface
```

Comment:
Why not just use `ResponseInterface` of [PSR-7](https://www.php-fig.org/psr/psr-7/)? :slightly_smiling_face:

Reply:
This code is moved from https://github.com/SocialConnect/common
Related to PSR-7 i will review and play with it a little bit before moving to it

Thanks

Reply:
Great. If you do decide to switch I suggest checking out [zend-diactoros](https://github.com/zendframework/zend-diactoros/) for PSR-7 compliant request/response classes.